demo.py

```diff
@@ -27,7 +27,10 @@
                          help='Specify the link that you want to add a tool slot too (exact link name from urdf)')
     optParser.add_option('--rotation', dest='initRotation', default='0 1 0 0',
                          help='Set the rotation field of your PROTO file.')
+    optParser.add_option('--init-pos', dest='initPos', default=None,
+                         help='Set the initial positions of your robot joints. Example: --init_pos="[1.2, 0.5, -1.5]" would set '
+                         ' the first 3 joits of your robot to the specified values, and leave the rest with their default value')
```

Comment:
These lines are too long (>128).

Reply:
ahh I was wondering what your definition is line length is. pep8 is 80 by default. Good to know, will keep that in mind for the future

Reply:
Yes, indeed, for this rule, we allow line a bit longer than default PEP8, for info, here is our Python coding style: https://github.com/cyberbotics/webots/wiki/Python-Coding-Style